boefjes/boefjes/plugins/kat_testssl_sh_ciphers/main.py

```diff
@@ -19,20 +22,29 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
     else:
         args = f" --jsonfile tmp/output.json --server-preference {address}:{ip_port}"
 
+    timeout = getenv("TIMEOUT", 30)
+
+    environment_vars = {
+        "OPENSSL_TIMEOUT": timeout,
+        "CONNECT_TIMEOUT": timeout,
+    }
+
     client = docker.from_env()
     container = client.containers.run(
         SSL_TEST_IMAGE,
         args,
         detach=True,
+        environment=environment_vars,
     )
 
-    container.wait()
-
-    output = get_file_from_container(container, "tmp/output.json")
-
     try:
+        container.wait(timeout=300)
+        output = get_file_from_container(container, "tmp/output.json")
+    except (APIError, RequestException) as e:
+        logging.warning("DockerException occurred: %s", e)
+        container.stop()
+        raise Exception("Timeout occurred while running testssl.sh")
```

Comment:
```suggestion
        raise Exception("Error occurred (possibly a timeout) while running testssl.sh")
```

The reason I am nitpicking about this is that in my experience software that gives hard to understand or incorrect error message makes it hard to figure out what goes wrong and can cause a lot of frustation when you use such software.